src/bootstrap/builder.rs

```diff
@@ -780,12 +780,7 @@ impl<'a> Builder<'a> {
             .arg("-Zconfig-profile");
 
         let profile_var = |name: &str| {
-            let profile = if self.config.rust_optimize {
-                "RELEASE"
-            } else {
-                "DEV"
-            };
-            format!("CARGO_PROFILE_{}_{}", profile, name)
+            format!("CARGO_PROFILE_RELEASE_{}", name)
```

Comment:
So I explicitly don't want to stop having this option internally, just want to drop it from the documented options.

Reply:
Sure! Will revert this.